examples/Browser/Server/Startup.cs

```diff
@@ -42,17 +55,42 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseHsts();
             }
 
-            app.UseHttpsRedirection();
+            // app.UseHttpsRedirection();
```

Comment:
Why did you comment this out?

Reply:
@JamesNK Otherwise it fails when calling non-ssl grpc endpoint